syncthing-git/PKGBUILD

```diff
@@ -5,63 +5,66 @@
 # Previous Maintainer : Sauyon Lee <sauyonl@sauyon.com>
 # Contributor : Martin Wimpress <code@flexion.org>
```

Comment:
please add me as Contributor

Reply:
I added you, if you want another format tell me here.